src/components/EditCoursePage/EditCourseForm.jsx

```diff
@@ -653,37 +659,33 @@ export class BaseEditCourseForm extends React.Component {
               extraInput={{ onInvalid: this.openCollapsible }}
               disabled={courseInReview}
             />
-            {entitlement && (
-              <React.Fragment>
-                <Field
-                  name="mode"
-                  component={RenderSelectField}
-                  label={
-                    <FieldLabel
-                      id="mode.label"
-                      text="Enrollment track"
-                      helpText={enrollmentHelp}
-                    />
-                  }
-                  extraInput={{ onInvalid: this.openCollapsible }}
-                  options={this.getEnrollmentTrackOptions()}
-                  disabled={courseInReview}
-                />
-                <Field
-                  name="price"
-                  component={RenderInputTextField}
-                  type="number"
-                  label={<FieldLabel text="Price" required requiredForSubmit />}
-                  extraInput={{
-                    onInvalid: this.openCollapsible,
-                    min: 0.01,
-                    step: 0.01,
-                  }}
-                  disabled={courseInReview}
-                  required={isSubmittingForReview}
+            <Field
+              name="mode"
+              component={RenderSelectField}
+              label={
+                <FieldLabel
+                  id="mode.label"
+                  text="Enrollment track"
+                  helpText={enrollmentHelp}
                 />
-              </React.Fragment>
-            )}
+              }
+              extraInput={{ onInvalid: this.openCollapsible }}
+              options={this.getEnrollmentTrackOptions()}
+              disabled={courseInReview || (!!entitlement && !!entitlement.sku)}
```

Comment:
Taking a hard stance in favor of `!!`, eh?

Reply:
I thought about not having it as `!!`, but disabled is expecting a boolean and that's the way of casting to bool. Otherwise I was running into scenarios where `entitlement = {}` and `entitlement.sku` evaluated to `null` so `disabled={null}` which felt worse.